internal/gengapic/genrest.go

```diff
@@ -112,7 +159,59 @@ func (g *generator) genRESTMethod(servName string, serv *descriptor.ServiceDescr
 	}
 }
 
+func (g *generator) emptyUnaryRESTCall(servName string, m *descriptor.MethodDescriptorProto) error {
+	info, err := getHTTPInfo(m)
+	if err != nil || info == nil {
+		return err
+	}
+
+	inType := g.descInfo.Type[*m.InputType]
+	inSpec, err := g.descInfo.ImportSpec(inType)
+	if err != nil {
+		return err
+	}
+
+	p := g.printf
+	lowcaseServName := lowcaseRestClientName(servName)
+	p("func (c *%s) %s(ctx context.Context, req *%s.%s, opts ...gax.CallOption) error {",
+		lowcaseServName, m.GetName(), inSpec.Name, inType.GetName())
+
+	// TODO(dovs): handle cancellation, metadata, osv.
+	// TODO(dovs): handle http headers
+	// TODO(dovs): handle deadlines?
+	// TODO(dovs): handle call options
+	p("// The default (false) for the other options are fine.")
+	p("m := protojson.MarshalOptions{AllowPartial: true, Multiline: true, EmitUnpopulated: true}")
+	p("if jsonReq, err := m.Marshal(req); err != nil {")
+	p("  return err")
+	p("}")
+	p("")
+	// TODO(dovs): handle path parameters
+	p("url := fmt.Sprintf(\"https://%%s%s\", c.host)", info.url)
```

Comment:
You can use backtick instead of `"` so that you don't have to escape the double quotes.

Reply:
Good to know! I'm not sure how the format string will interact with the backtick, though. I'll try it out and see how it works.

Reply:
To be clear, i mean you can do 
```go
p(`url := fmt.Sprintf("https://%%s%s", c.host)`, info.url)
```

Realized that my comment wasn't super clear.